packages/usa-range/src/styles/_usa-range.scss

```diff
@@ -22,7 +22,7 @@
   cursor: pointer;
 
   @media (forced-colors: active) {
-    outline: 2px solid transparent;
+    border: $border-high-contrast;
```

Comment:
I noticed there was a visual regression here. Is it possible to switch it back to outline?

![image](https://user-images.githubusercontent.com/93996430/226700935-53f3aa1d-fba5-4cd7-913e-cc4a50451e96.png)